shardingsphere-kernel/shardingsphere-transaction/shardingsphere-transaction-core/src/main/java/org/apache/shardingsphere/transaction/context/TransactionContextsBuilder.java

```diff
@@ -61,4 +74,99 @@ private TransactionRule getTransactionRule() {
         Optional<TransactionRule> transactionRule = globalRules.stream().filter(each -> each instanceof TransactionRule).map(each -> (TransactionRule) each).findFirst();
         return transactionRule.orElseGet(() -> new TransactionRule(new DefaultTransactionRuleConfigurationBuilder().build()));
     }
+    
+    private void generateNarayanaConfig(final TransactionRule transactionRule) {
```

Comment:
The Narayana impl still is `TransactionContextsBuilder` which should move out of common module